app/utils/callback/lava.py

```diff
@@ -453,6 +453,9 @@ def _add_test_results(group, suite_results, suite_name):
                 "value": float(measurement),
                 "unit": test["unit"],
             }]
+        reference = test.get("reference")
+        if reference and reference != 'None':
```

Comment:
`if reference` is enough, as it will do the same if it's `None` or an empty string.